src/Meshes/mesh.ts

```diff
@@ -884,8 +898,18 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
         }
 
         var distanceToCamera = bSphere.centerWorld.subtract(camera.globalPosition).length();
+        const useScreenCoverage = internalDataInfo._useLODScreenCoverage;
+        let ratio = 1;
+
+        if (useScreenCoverage) {

```

Comment:
could we here keep ratio in a variable compareValue. and in the else part make it = -distanceToCamera ?

So that in the 2 following comparison we do not need the extra if cases ???